opengever/disposition/reports.py

```diff
@@ -0,0 +1,169 @@
+        'schutzfrist',
+        'Ablage_Pr\xe4fix',
+        'Ablage_Nr',
+        'Beschreibung',
+    ]
+
+    def __init__(self, dossiers):
+        self.dossiers = dossiers
+
+    def __call__(self):
+        lines = []
+        for dossier in self.dossiers:
+            binding = dossier.binding()
+            lines.append(
+                {
+                    'dossier_id': binding.id,
+                    'Mandant': get_current_admin_unit().label(),
+                    'Ordnungsystem_Version': dossier.parents()[0].version,
+                    'Ordnungssystem_Pfad': self.repository_path(dossier),
+                    'Dossier_Titel': dossier.obj.title,
+                    'entstehungszeitraum_von': format_date(
+                        binding.entstehungszeitraum.von.datum),
+                    'entstehungszeitraum_bis': format_date(
+                        binding.entstehungszeitraum.bis.datum),
+                    'klassifizierungskategorie': binding.klassifizierungskategorie,
+                    'datenschutz': binding.datenschutz,
+                    'oeffentlichkeitsstatus': binding.oeffentlichkeitsstatus,
+                    'aktenzeichen': binding.aktenzeichen,
+                    'eroeffnungsdatum': format_date(dossier.binding().eroeffnungsdatum.datum),
+                    'abschlussdatum': format_date(dossier.binding().abschlussdatum.datum),
+                    'schutzfrist': binding.schutzfrist,
+                    'Ablage_Pr\xe4fix': dossier.obj.filing_prefix,
+                    'Ablage_Nr': self.filing_number(dossier),
+                    'Beschreibung': dossier.obj.description,
+                })
+
+        return lines
+
+    def repository_path(self, dossier):
+        # skip dossier itself
+        return u'/'.join(
+            [parent.Title().decode('utf-8') for parent in dossier.parents()[:-1]])
+
+    def filing_number(self, dossier):
+        if IFilingNumberMarker.providedBy(dossier.obj):
+            return IFilingNumber(dossier.obj).filing_no
+
+
+class DispositionDocumentCSVReporter(object):
+
+    fieldnames = [
+        'dossier_id',
+        'document_id',
+        'laufnummer',
+        'sip_folder_name',
+        'sip_file_name',
+        'document_title',
+        'erscheinungsform',
+        'registrierdatum',
+        'entstehtungszeitraum_von',
+        'entstehtungszeitraum_bis',
+        'klassifizierungskategorie',
+        'datenschutz',
+        'oeffentlichkeitsstatus',
+        'oeffentlichkeitsstatusBegruendung',
+        'dateiRef',
+        'originalName',
+        'pruefalgorithmus',
+        'pruefsumme',
+        'dokumentdatum',
+        'beschreibung',
+        'autor',
+        'aktenzeichen']
+
+    def __init__(self, dossiers):
+        self.dossiers = dossiers
+
+    def __call__(self):
+        lines = []
+        for dossier in self.dossiers:
+            for key, document in dossier.documents.items():
+                for _file in document.files:
```

Comment:
The underscore prefix is meant for private/internal variables. To augment reserved keyword usually a suffix is used.
```suggestion
                for file_ in document.files:
```